Bermuda/Bermuda/Rabbit.cpp

```diff
@@ -24,34 +25,50 @@ MovableEntity(id, spawnPoint->getX(), spawnPoint->getY())
 	this->movingRight = false;
 	this->movingDown = false;
 	this->movingUp = false;
-	//this->interaction = false;
+	#pragma endregion Moving_stuff
```

Comment:
De "#pragma endregion" heeft geen naam nodig. Is alleen nodig bij "pragma region".